src/background/api/index.js

```diff
@@ -186,4 +187,35 @@ export async function getRuntimeNameList(){
   }
   let nonce = account?.general?.nonce || 0
   return { balance, nonce,allowanceList }
+}
+
+/**
+ * get runtime balance and allowance
+ * @param {*} address 
+ * @param {*} runtimeId 
+ * @returns 
+ */
+ export async function getRuntimeBalance(address,runtimeId,propDecimals){ 
+  const oasisClient = getOasisClient()
+  const CONSENSUS_RT_ID = oasis.misc.fromHex(runtimeId)
+  const accountsWrapper = new oasisRT.accounts.Wrapper(CONSENSUS_RT_ID);
+  const balancesResult = await accountsWrapper
+      .queryBalances()
+      .setArgs({
+          address: await oasis.staking.addressFromBech32(address),
+      })
+      .query(oasisClient).catch(err=>{return err});
+  let nativeDenominationBalanceBI = 0n;
+  if(balancesResult.balances){
+    const nativeDenominationHex = oasis.misc.toHex(oasisRT.token.NATIVE_DENOMINATION);
+    for (const [denomination, amount] of balancesResult.balances) {
+        const denominationHex = oasis.misc.toHex(denomination);
+        if (denominationHex === nativeDenominationHex) {
+            nativeDenominationBalanceBI = oasis.quantity.toBigInt(amount);
+            break;
+        }
+    }
+  }
+  let decimals = propDecimals||9
```

Comment:
if propDecimals = 0, then decimals should = 0
```suggestion
  const decimals = propDecimals ?? 9
```

Reply:
```suggestion
  lconst decimals = propDecimals ??  cointypes.decimals
```

Reply:
better , i will fix this

Reply:
there i commit twice , because i find  `""` will pass in  `??` . so we need check is propDecimals a number .
the last commit is [this](https://github.com/oasisprotocol/oasis-wallet-ext/pull/168/commits/deaecedcd502df8d180ccbeb47b72d8d353cd194)